panther_analysis_tool/main.py

```diff
@@ -230,6 +233,11 @@ def load_analysis_specs(directories: List[str]) -> Iterator[Tuple[str, str, Any,
                 # skip loading files that have already been imported
                 if spec_filename in loaded_specs:
                     continue
+                # Dont load files that are explictly ignored
```

Comment:
As a note for a potential future enhancement, you could consider splitting the file discovery logic from the YAML parsing logic.

Also, I just noticed that we handle JSON files here. Normally YAML parsers can handle the JSON format, because YAML is a superset of JSON (again just a note for future consideration, completely unrelated to this PR 😄 ).